integration-test.Rmd

```diff
@@ -26,15 +26,18 @@ knitr::opts_chunk$set(
 Packages used in this file:
 
 ```{r}
-suppressPackageStartupMessages(library(tidyverse))
-suppressPackageStartupMessages(library(devtools))
-suppressPackageStartupMessages(library(testthat))
-suppressPackageStartupMessages(library(config))
-suppressPackageStartupMessages(library(rlang))
-suppressPackageStartupMessages(library(renv))
-suppressPackageStartupMessages(library(glue))
-suppressPackageStartupMessages(library(fs))
+library(tidyverse)
+library(devtools)
+library(testthat)
+library(config)
+library(rlang)
+library(renv)
+library(glue)
+library(fs)
```

Comment:
I no longer suppress package startup messages. The output is long but informative.